solutions/src/test/java/ru/sberbank/school/task09/CacheRouteServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package ru.sberbank.school.task09;
+
+import org.junit.jupiter.api.*;
+import ru.sberbank.school.task09.util.KryoManager;
+import ru.sberbank.school.task09.util.SerializeManager;
+
+import java.io.File;
+
+
+class CacheRouteServiceTest {
+    private static CacheRouteService cacheRouteService;
+    private static String directoryPath;
+
+    @BeforeAll
+    static void init() {
+        directoryPath = "C:\\Users\\1357028\\Desktop\\Save";
+        cacheRouteService = new CacheRouteService(directoryPath, new KryoManager(directoryPath));
+    }
+
+    @AfterEach
+    void deleteFile () {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2093215097>*